server/src/internalClusterTest/java/org/opensearch/action/admin/indices/datastream/DataStreamTestCase.java

```diff
@@ -110,4 +126,7 @@ public AcknowledgedResponse deleteIndexTemplate(String name) throws Exception {
         return response;
     }
 
+    protected boolean performRemoteStateRestore() {
```

Comment:
this as well

Reply:
triggerRemoteStateRestore, Also add comments

Reply:
Renamed and moved it to `OpenSearchIntegTestCase`.